packages/cdc-react/src/components/Modal/Modal.tsx

```diff
@@ -0,0 +1,69 @@
+import "./Modal.scss";
+import { Button } from "@us-gov-cdc/cdc-react";
+import { Icons } from "@us-gov-cdc/cdc-react-icons";
+import { useRef } from "react";
+export interface ModalProps {
+  isOpen: boolean;
+  onClose: () => void;
+  children?: React.ReactNode;
+  isForcedAction?: boolean;
+}
+export interface ModalChildrenProps {
+  children: React.ReactNode;
+}
+
+export const Modal = (props: ModalProps) => {
+  const overlayRef = useRef<HTMLDivElement>(null);
+  const modalRef = useRef<HTMLDivElement>(null);
+
+  const handleOverlayClick = (event: React.MouseEvent<HTMLDivElement>) => {
+    if (event.target === overlayRef.current && !props.isForcedAction) {
+      props.onClose();
+    }
+  };
+
+  if (!props.isOpen) return null;
+
+  return (
+    <div
+      ref={overlayRef}
+      onClick={handleOverlayClick}
+      role="presentation"
+      className="modal-overlay">
+      <div
+        ref={modalRef}
+        className="modal usa-modal"
+        role="dialog"
+        aria-modal="true">
+        <div className="modal-content usa-modal__content">
+          <div className="modal-main">{props.children}</div>
+          <Button
+            ariaLabel="Close this window"
+            size="tiny"
+            iconOnly={true}
+            icon={<Icons.Close />}
+            variation="unstyled"
+            className="usa-button usa-modal__close modal-close-btn"
+            onClick={props.onClose}></Button>
+        </div>
+      </div>
+    </div>
+  );
+};
+
+// TODO: extract these components to their own files
+export const ModalTitle = (props: ModalChildrenProps) => {
```

Comment:
I don't think the `ModalTitle` component is necessary since we aren't (and don't have the requirement to) giving the user the ability to change the look and feel of the title.  They are really just passing a string that'll be rendered in an `h2`.  I feel like a prop on the `Modal` component is better suited for this.  You can also couple the "X" close button with the modal header layout, which seems more intuitive.

Reply:
I can remove the modal title component and make it a prop. In terms of the "x" close button, I followed the same build as the USWDS modals. Abbey mentioned how the last element in the modal is the close button. This has to do with accessibility and tabbing which is to be implemented later.